src/pdfkit.cs

```diff
@@ -1233,9 +1232,11 @@ interface PdfDocument : NSCopying {
 		[Export ("minorVersion")]
 		nint MinorVersion { get; }
 #else
+		[NoiOS]
 		[Export ("majorVersion")]
 		int MajorVersion { get; } /* int, not NSInteger */
 
+		[NoiOS]
```

Comment:
Do we need this at all? I mean, we already have the #if IOS and AFAIK @chamons  tool won't care.

Reply:
No I don't think it is necessary, but I think @chamons said to mark as much as we can when we can't remove defines